modules/system/console/OctoberInstall.php

```diff
@@ -130,6 +131,25 @@ protected function setupAdvancedValues()
         $this->writeToConfig('app', ['debug' => $debug]);
     }
 
+    protected function askToInstallPlugins() {
+        $drivers = (bool)$this->confirm('Do you wish to install October.Drivers plugin?', false);
```

Comment:
Conversion to bool should not be required here, `$this->confirm()` already [returns bool](https://github.com/laravel/framework/blob/5.4/src/Illuminate/Console/Command.php#L294)